examples/all-clusters-app/all-clusters-common/src/bridged-actions-stub.cpp

```diff
@@ -37,7 +37,7 @@ class BridgedActionsAttrAccess : public AttributeAccessInterface
     // Register for the Bridged Actions cluster on all endpoints.
     BridgedActionsAttrAccess() : AttributeAccessInterface(Optional<EndpointId>::Missing(), BridgedActions::Id) {}
 
-    CHIP_ERROR Read(const ConcreteReadAttributePath & aPath, AttributeValueEncoder & aEncoder) override;
+    CHIP_ERROR Read(FabricIndex fabricIndex, const ConcreteReadAttributePath & aPath, AttributeValueEncoder & aEncoder) override;
```

Comment:
This change should not have been made.  The fabric index was already available as `aEncoder.AccessingFabricIndex()`.